rmgpy/rmg/mainTest.py

```diff
@@ -151,7 +152,19 @@ def testRMGSeedWorks(self):
         self.rmg.reactionModel.addSeedMechanismToCore('testSeed_edge')  # try adding seed as seed mech
         self.assertTrue(len(self.rmg.reactionModel.core.species) > 0)
         self.assertTrue(len(self.rmg.reactionModel.core.reactions) > 0)
-
+    
+    def testRMGMemory(self):
+        """
+        test that RMG Memory objects function properly
+        """
+        for rxnsys in self.rmg.reactionSystems:
+            Rmem = RMG_Memory(rxnsys)
+            Rmem.generateCond()
+            Rmem.getCond()
+            Rmem.addtConvN(1.0,.2,2)
+            Rmem.generateCond()
+            Rmem.getCond()
```

Comment:
Perhaps add relevant assertions and helpful error messages?

Reply:
I couldn't really think of anything I felt the need to specifically check in the object other than that the basic operations could be done.  It is deterministic so I could hard code the test (check whether its generating the same condition), but I feel like unless I'm checking something I have reason to worry about that just forces someone to change the hard coded numbers anytime they change something in the weighing or optimization algorithm.